dev-tools/omdb/src/bin/omdb/nexus.rs

```diff
@@ -1090,6 +1106,24 @@ async fn cmd_nexus_blueprints_regenerate(
     Ok(())
 }
 
+async fn cmd_nexus_blueprints_import(
+    client: &nexus_client::Client,
+    _destruction_token: DestructiveOperationToken,
+    args: &BlueprintImportArgs,
+) -> Result<(), anyhow::Error> {
+    let input_path = &args.input;
+    let file = std::fs::File::open(input_path)
+        .with_context(|| format!("open {:?}", input_path))?;
+    let blueprint: Blueprint = serde_json::from_reader(file)
+        .with_context(|| format!("read {:?}", input_path))?;
+    client
+        .blueprint_import(&blueprint)
+        .await
+        .with_context(|| format!("upload {:?}", input_path))?;
```

Comment:
Since it's a `Utf8PathBuf`, you can just use the `Display` impl.

```suggestion
        .with_context(|| format!("open {}", input_path))?;
    let blueprint: Blueprint = serde_json::from_reader(file)
        .with_context(|| format!("read {}", input_path))?;
    client
        .blueprint_import(&blueprint)
        .await
        .with_context(|| format!("upload {}", input_path))?;
```

Also, `serde_json::from_reader` will read one byte at a time -- worth throwing a `BufReader` around it to avoid tons of syscalls, or just read the file into a string.